tretools/utility/custom_phen_conversion.py

```
@@ -0,0 +1,48 @@
+import os
+import csv
+
+def process_custom_codelist(input_file, output_directory, output_prefix, code_col_name='term', ICD10_col_name='ICD10', SNOMED_col_name='SNOMED', OPCS4_col_name='OPCS4'):
```

This function has 7 inputs. This makes it harder to read and makes it very easy to place in the wrong order, which may cause it to error, or worse a silence error that we only find later. It would be best to pass the col names in as a config dictionary. 

I am not sure we need the output_prefix.

---

A config dictionary is used now.